src/api/requests.ts

```diff
@@ -1,60 +1,66 @@
+import { IKartLapsPerRun, ILapDataset } from "../models/go-kart-types";
+
 // Get all sessions
-export default function getAllRuns(callback, errorCallback, finallyCallback) {
+export default function getAllRuns(
+  callback: (allRunsJSON: string[]) => void,
+  errorCallback: (error: Error) => void,
+  finallyCallback: (() => void) | undefined = undefined
```

Comment:
I believe finallyCallback is an optional field as you later use an `if (finallyCallback)`, if so you could add a ? after the parameter to make it optional.

Reply:
I learnt a new thing. This was my bad.
So then the parameter can just be:
`finallyCallback?: () => void`